metacity/meson.build

```diff
@@ -0,0 +1,11 @@
+flavours = []
+foreach flavour: ['default', 'dark', 'light']
```

Comment:
It would be good if we can find a way to define this "flavour" list in one place only (not sure it is possible), otherwise keep a note about update-it/keep-it-in-sync if we want to add a new GTK Flavour (the other place where this list exists is gtk/meson.build)

Reply:
Yeah, I've just copied the pattern I've seen used elsewhere.